<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>client</class>
 <widget class="QDialog" name="client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>834</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>931</width>
     <height>611</height>
    </rect>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_2">
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <attribute name="title">
     <string>ajouter</string>
    </attribute>
    <widget class="QPushButton" name="pb_supprimer">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>470</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_modifier">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>470</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_ajouter">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>300</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="trie_age">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>60</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Trier selon age</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>140</y>
       <width>591</width>
       <height>311</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>589</width>
        <height>309</height>
       </rect>
      </property>
      <widget class="QTableView" name="tabemploye">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>591</width>
         <height>311</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QLabel" name="type">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>adresse</string>
     </property>
    </widget>
    <widget class="QLabel" name="nom">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Nom </string>
     </property>
    </widget>
    <widget class="QLabel" name="prenom">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Prenom</string>
     </property>
    </widget>
    <widget class="QLabel" name="age">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Age</string>
     </property>
    </widget>
    <widget class="QLabel" name="matricule">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_matricule">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>110</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_nom">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>200</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_prenom">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>140</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_age">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>230</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_type">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>170</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="recherche">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>90</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>470</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>num telephone</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_matriculd">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>260</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>-50</x>
       <y>-15</y>
       <width>1041</width>
       <height>611</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../Downloads/charte essai 1.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="trie_nom">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>100</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Trier nom</string>
     </property>
    </widget>
    <widget class="QPushButton" name="trie_prenom">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>20</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Trier prenom</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_statistics">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>470</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>stats</string>
     </property>
    </widget>
    <zorder>label</zorder>
    <zorder>pb_supprimer</zorder>
    <zorder>pb_modifier</zorder>
    <zorder>pb_ajouter</zorder>
    <zorder>trie_age</zorder>
    <zorder>scrollArea</zorder>
    <zorder>type</zorder>
    <zorder>nom</zorder>
    <zorder>prenom</zorder>
    <zorder>age</zorder>
    <zorder>matricule</zorder>
    <zorder>le_matricule</zorder>
    <zorder>le_nom</zorder>
    <zorder>le_prenom</zorder>
    <zorder>le_age</zorder>
    <zorder>le_type</zorder>
    <zorder>recherche</zorder>
    <zorder>comboBox</zorder>
    <zorder>label_9</zorder>
    <zorder>le_matriculd</zorder>
    <zorder>trie_nom</zorder>
    <zorder>trie_prenom</zorder>
    <zorder>pb_statistics</zorder>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>statistique</string>
    </attribute>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>3</y>
       <width>941</width>
       <height>481</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../Bureau/247590809_624209822325987_6344712974586411864_n.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-10</y>
       <width>941</width>
       <height>601</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../Downloads/charte essai 1.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QFrame" name="horizontal_stat">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>651</width>
       <height>471</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout"/>
    </widget>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>horizontal_stat</zorder>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>email</string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit_psswrd_email">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>80</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="placeholderText">
      <string>password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_browse_email">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>140</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse...</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_email">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>771</width>
       <height>201</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_from_email">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>331</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="placeholderText">
      <string>from</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_send_email">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>170</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>send</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_atchmnt_email">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>491</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="placeholderText">
      <string>attachment</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_subject_email">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>491</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="placeholderText">
      <string>subject</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_to_email">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>591</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="placeholderText">
      <string>to</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Page</string>
    </attribute>
    <widget class="QPushButton" name="abrir">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>abrir </string>
     </property>
    </widget>
    <widget class="QPushButton" name="pause">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>50</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>pause</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stop">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>50</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>stop</string>
     </property>
    </widget>
    <widget class="QPushButton" name="play">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>play</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="avance">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>381</width>
       <height>21</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QSlider" name="volum">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>150</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::NoTicks</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="mute">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>150</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>mute </string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
